packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -66,15 +70,18 @@ describe('Quick launch apex tests', () => {
       .withArgs('defaultusername')
       .returns(testData.username);
     notificationServiceStub = sb.stub(notificationService, 'showErrorMessage');
-    sb.stub(workspaceContext, 'getConnection')
-      .returns(mockConnection);
+    sb.stub(workspaceContext, 'getConnection').returns(mockConnection);
     testServiceStub = sb
       .stub(TestService.prototype, 'runTestSynchronous')
-      .resolves({ tests: [{ apexLogId: APEX_LOG_ID }] } as TestResult);
+      .resolves({ tests: [{ apexLogId: APEX_LOG_ID }] });
     buildPayloadStub = sb.stub(TestService.prototype, 'buildSyncPayload');
     writeResultFilesStub = sb.stub(TestService.prototype, 'writeResultFiles');
-    createCheckpointStub = sb.stub(breakpoints, 'sfdxCreateCheckpoints');
-    oneOrMoreActiveCheckpointsStub = sb.stub(CheckpointService.prototype, 'hasOneOrMoreActiveCheckpoints');
+    createCheckpointStub = sb.stub(CheckpointService, 'sfdxCreateCheckpoints');
```

Comment:
This stub required moving the `sfdxCreateCheckpoints` method to a static method in the class ( or I could have wrapped it in an object to export but since it was stateless a static method on the already exported class seemed cleaner).